### ISSUE TITLE
Sampling-free obstacle gradients and reactive planning in Neural Radiance Fields (NeRF). (arXiv:2205.01389v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#18566]
Sampling-free obstacle gradients and reactive planning in Neural Radiance Fields (NeRF). (arXiv:2205.01389v1 [cs.RO])<br>
https://ift.tt/KBZxYaI<br>
<p>This work investigates the use of Neural implicit representations, specifically Neural Radiance Fields (NeRF), for geometrical queries and motion planning. We show that by adding the capacity to infer occupancy in a radius to a pre-trained NeRF, we are effectively learning an approximation to a Euclidean Signed Distance Field (ESDF). Using backward differentiation of the augmented network, we obtain an obstacle gradient that is integrated into an obstacle avoidance policy based on the Riemannian Motion Policies (RMP) framework. Thus, our findings allow for very fast sampling-free obstacle avoidance planning in the implicit representation.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
